### ISSUE TITLE
"now.tz is not a function" when MMM-Powerwall below default clock module in config.js

### ISSUE THREAD

**Issue by daterrell** [MikeBishop/MMM-Powerwall#67]
I noticed strange behavior, the clock and calendar I normally keep in the upper-left were no longer loading.  I'm sorry, I can't say if this was a recent change with the MagicMirror repo changes, the clock module hasn't been updated for 4 months (and still uses moment and moment-timezone as it did before).  With MMM-Powerwall module lower in my config.js, it somehow causes moment/moment-timezone to 'unload' (I don't have a better way to describe this, I'm not a web dev).  Moving Powerwall above the clock in the config lets it load the required moment-timezone module.

**Comment by daterrell** [comment]
Also, I'm far from certain this is this module's 'fault', I just know what the error is, I know it happens with this module, and when it's below the clock module in the config.

**Comment by MikeBishop** [comment]
That is a fascinating interaction.  As part of the move to ChartJS v3, I needed to take an explicit dependency on moment.js.  I wonder if we're using different versions and somehow stomping the version that clock expects?

#68 points to the system copy the same way the clock module does; hopefully that will solve this for you.  Once you give it a try, I'll merge it.  (`git fetch` followed by `git checkout of-no-moment`)

**Comment by daterrell** [comment]
It is _really_ strange how that happened.  And this change does fix the issue.

Incidentally, the few modules I wrote also use `moment.js` and I opted to copy how MM requires it.  I'm not sure why they're holding onto Moment.  I noticed it was deprecated as a whole, and tried to find something to replace it, eventually deciding that MM will either hold onto Moment (and I'll continue to use it as well), or they'll migrate to something else and I'll follow suit.
